charts/asset-manager/templates/podMonitor.yaml

```diff
@@ -7,13 +7,9 @@ metadata:
     app.kubernetes.io/name: prometheus-operator
     app.kubernetes.io/part-of: kube-prometheus
   name: {{ .Release.Name }}
-  namespace: monitoring
 spec:
   selector:
     matchLabels:
       app: {{ .Release.Name }}
-  namespaceSelector: 
```

Comment:
Is the default behaviour equivalent to this (i.e. only match stuff in the apps namespace because that's the namespace that the PodMonitor's in), or does it default to matching all namespaces? If it's all namespaces then that's problematic because it means the metrics will get polluted if there's ever an app with the same name running in another namespace.

Reply:
(I tried to look this up, but the API docs for namespaceSelector weren't very helpful on this point.)

Reply:
I think it is ok, based one: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#namespaceselector